osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -194,6 +199,8 @@ private void load(BeatmapManager beatmaps, AudioManager audio, DialogOverlay dia
             if (this.beatmaps == null)
                 this.beatmaps = beatmaps;
 
+            rulesetConversionAllowed = config.GetBindable<bool>(OsuSetting.ShowConvertedBeatmaps);

```

Comment:
i think we want to bind `rulesetConversionAllowed.ValueChanged += ensurePlayableRuleset` too, right?

Reply:
the carousel itself seemed to handle it by changing filters